apps/editors/helpers.py

```diff
@@ -220,6 +221,29 @@ def reviewers_score_bar(context, types=None, addon_type=None):
                                          addon_type=addon_type)))
 
 
+@register.inclusion_tag('editors/includes/files_view.html')
+@jinja2.contextfunction
+def all_distinct_files(context, version):
+    """Only display a file once even if it's been uploaded
+    for several platforms."""
+    # hashes_to_file will group files per hash:
+    # {<file.hash>: [<file>, 'Windows / Mac OS X']}
+    hashes_to_file = {}
+    for file_ in version.all_files:
+        display_name = force_text(amo.PLATFORMS[file_.platform].name)
+        if file_.hash in hashes_to_file:
+            hashes_to_file[file_.hash][1] += ' / ' + display_name
+        else:
+            hashes_to_file[file_.hash] = [file_, display_name]
+    return new_context(dict(
```

Comment:
You don't need `new_context` here. Either drop the `new_context`, or drop the `dict` so you just have `new_context(context, distinct_files=hashes_to_file.values(), version=version)`